### PR TITLE
Remove all non-deterministic data from visual tests

### PR DESCRIPTION
**Pull request by callumacrae**
### Context

Will fix the percy errors in every PR 😬

### CODE REVIEW

File: spec/cypress/app_commands/scenarios/admin/schools.rb [hunk 1]
```diff
@@ -3,4 +3,7 @@
 school = FactoryBot.create(:school, name: "Include this school", urn: 123_456)
 FactoryBot.create(:user, :induction_coordinator, schools: [school])
 
+Faker::UniqueGenerator.clear
+Faker::Config.random = Random.new(42)
```
Comment (ethanmills) [inline spec/cypress/app_commands/scenarios/admin/schools.rb:7]: The most random of numbers

**Comment by kicferk1** [comment]
Well, this is probably a good makeshift solution, but I really think in the long run we should re-consider having any randomness to begin with in cypress, so that we don't re-introduce this problem in the future :/

**Review by ethanmills — APPROVED** [review]

**Comment by callumacrae** [comment]
> Well, this is probably a good makeshift solution, but I really think in the long run we should re-consider having any randomness to begin with in cypress, so that we don't re-introduce this problem in the future :/

Yep, it's definitely something to look out for in future PRs. Not sure of a way of automatically doing it short of removing faker from all the factories, though

**Pull request merged**